models/turbine_models/custom_models/sd_inference/sd_pipeline.py

```diff
@@ -704,10 +843,35 @@ def numpy_to_pil_image(images):
     from turbine_models.custom_models.sd_inference.sd_cmd_opts import args
 
     ireec_flags = {
-        "clip": args.ireec_flags + args.clip_flags,
+        "text_encoder": args.ireec_flags + args.clip_flags,
         "scheduler": args.ireec_flags,
         "unet": args.ireec_flags + args.unet_flags,
-        "vae_decode": args.ireec_flags + args.vae_flags,
+        "mmdit": args.ireec_flags + args.mmdit_flags,
+        "vae": args.ireec_flags + args.vae_flags,
+    }
+    devices = {
```

Comment:
Looks like we have to specify every submodels device? Necessary for compiling/running on multiple devices but can we have an overall optional flag to override and set all of these once if we are only running on one device?

Reply:
the pipeline does handle a string input for device and iree target. It's not really the best way to manage it at this point -- I'd rather the API consumer explicitly define targets for each submodel, but both modes work both in the CLI parser and the pipeline API.